prepare-debian-ubuntu.sh

```diff
@@ -9,3 +12,7 @@ sudo apt-get install $@ g++ build-essential autoconf automake automake1.9 cmake
 # instead of the intended program.
 sudo true; echo "dash dash/sh boolean false" | sudo debconf-set-selections
 sudo dpkg-reconfigure --frontend=noninteractive dash
+
+# Make Type-B PSP devices (eg. PSPs running PSPLink) mount world-writable,
+# removing the need to run usbhostfs_pc as root. See above for `sudo true`.
+sudo true; sudo cp dist/99-psplink.rules /etc/udev/rules.d/99-psplink.rules
```

Comment:
Do you think we should reload udev rules with `udevadm` and trigger them here as well in case a PSP is already connected?